app/routes/v2/orgs.js

```diff
@@ -41,22 +41,20 @@ const createOrg = async(req, res) => {
     return res.status(400).send( 'This org already exists' );
   }
 
-  const orgAdminKey = req.orgAdminKey; // this was set in verifyAdminOrgKey()
   const orgApiKey = `orgApiKey-${uuid()}`;
   try {
     const insertedOrg = await Orgs.insertOne({
       '_id': uuid(),
       'name': orgName,
       'orgKeys' : [ orgApiKey ],
-      'orgAdminKey': orgAdminKey,
       'created': new Date(),
       'updated': new Date()
     });
-
+   
     if(insertedOrg.result.ok) {
       return res.status(200).send( insertedOrg.ops[0] );
     } else {
-      req.log.error(insertedOrg);
+      req.log.error(`${res.statusCode}  Could not create ${orgName}`);
```

Comment:
Should have made this suggestion more explicitly the first time. I think you want 
req.log.error(insertedOrg.result, `Failed to insert org ${orgname} document into Orgs collection.`);